app/styles/partials/all.scss

```diff
@@ -14,3 +14,4 @@
 @import 'speaker';
 @import 'rails';
 @import 'visibility-helpers';
+@import "x-scrollable";
```

Comment:
Why would you create a stylesheet just for this ? it doesn't make sense does it since we can't use it for anything else ? Create a stylesheet maybe named as `utils`. And also follow the style guide at `docs/style-guide.md`